### PR TITLE
ARROW-3664: [Rust] Add benchmark for PrimitiveArrayBuilder

### PR DESCRIPTION
**Pull request by sunchao**

### CODE REVIEW

File: ci/travis_script_rust.sh [hunk 1]
```diff
@@ -35,6 +35,7 @@ cargo rustc -- -D warnings
 
 cargo build
 cargo test
+cargo bench
```
Comment (kszucs) [inline ci/travis_script_rust.sh]: It takes about a minute to run, so for now it's acceptable to run benchmarks on CI.

**Review by kszucs — APPROVED** [review]
LGTM

File: ci/travis_script_rust.sh (hunk 1 above)
Comment (sunchao) [reply ci/travis_script_rust.sh]: Yes. We can cut it further down if it's needed. BTW: are we running benchmarks for other languages at the moment?

Comment (kszucs) [reply ci/travis_script_rust.sh]: AFAIK no

**Review by paddyhoran — APPROVED** [review]
LGTM, thanks @sunchao

**Comment by sunchao** [comment]
Thanks @kszucs and @paddyhoran !

**Pull request closed without merging**